celery_worker/tests/test_tasks.py

```
@@ -0,0 +1,61 @@
+
+        assert result["status"] in ["completed", "success"]
+
+    @patch("worker.get_drawing_sync")
+    def test_process_drawing_handles_missing_drawing(self, mock_get_drawing):
+        """Чертёж не найден → задача возвращает ошибку"""
+        mock_get_drawing.return_value = None
+
+        result = process_drawing("nonexistent", "Вопрос")
+
+        assert result["status"] in ["failed", "completed"]
+        # В реальном коде при ошибке поле error может отсутствовать
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `В` (CYRILLIC CAPITAL LETTER VE). Did you mean `B` (LATIN CAPITAL LETTER B)?

(RUF003)